src/chat.rs

```diff
@@ -3777,15 +3779,6 @@ pub(crate) async fn add_contact_to_chat_ex(
             return Err(e);
         }
         sync = Nosync;
-        if sync_qr_code_tokens
```

Comment:
Actually this reverts the fix trying to sync QR code tokens always after promoting groups, so the user should upgrade all devices to avoid problems with tokens synchronisation. Anyway that fix doesn't help if we merge #5901

Reply:
Reverted this. Let's not increase the diff and not create merge conflicts with #5901